app/views/admin/secrets/edit.html.erb

```diff
@@ -1,16 +1,46 @@
+<%= javascript_include_tag "/deploy_groups.js" %>
```

Comment:
w/o it generates a path that resovles to public/javascripts/$file (which we didn't have, but now down fyi).